### ISSUE TITLE
MangaOwl: no pages found 

### ISSUE THREAD

**Issue by D1012** [tachiyomiorg/tachiyomi-extensions#10464]
### Source information

MangaOwl 1.2.24

### Steps to reproduce

No pages found

### Expected behavior

The pages should open 

### Actual behavior

No pages found
![Screenshot_20220115_113016_eu kanade tachiyomi j2k](https://user-images.githubusercontent.com/84666471/149607331-81ec0784-8984-443b-8aec-100bb438b249.jpg)

### Tachiyomi version

v1.4.4

### Android version

Android 9

### Other details

It was okay this morning but I was just to read some manga then this happened again. 
![Screenshot_20220115_113016_eu kanade tachiyomi j2k](https://user-images.githubusercontent.com/84666471/149607368-cd65eaeb-4fda-455b-90a3-ffb755564af1.jpg)

### Acknowledgements

- [X] I have searched the existing issues and this is a new ticket, **NOT** a duplicate or related to another open issue.
- [X] I have written a short but informative title.
- [X] I have updated the app to version **[0.12.3](https://github.com/tachiyomiorg/tachiyomi/releases/latest)**.
- [X] I have updated all installed extensions.
- [X] I have tried the [troubleshooting guide](https://tachiyomi.org/help/guides/troubleshooting/).
- [X] If this is an issue with the app itself, I should be opening an issue in the [app repository](https://github.com/tachiyomiorg/tachiyomi/issues/new/choose).
- [X] I will fill out all of the requested information in this form.

**Comment by mireithemage** [comment]
I believe I have found the solution for both the no page error and the http/https error. I spent a few hours testing before I had a conclusive answer. The problem is a corruption in the data of the chapters in the library interfering with the correct online data. If you check any manga from MangaOwl that have never been in your library before the errors happened, you will find there are no errors. Simply removing it from your library and adding it back will not work (because the data is still stored on your device). If you have only a few titles from MangaOwl, follow the "Short Solution". If you have a lot of manga (like I do), follow the "Big Library Solution".
---------------------
Short Solution:
1) Copy (or memorize) the name of the manga you want fixed
2) Remove it (and any downloaded chapters) from your library
3) Go to Settings > Advanced and tap on "Clear database".
4) Find the manga again on MangaOwl
5) Test to see it work and re-add to library
6) Repeat until done
‐----------------------
Big Library Solution:
1) Back up your library EXCEPT for "Chapters". You don't want to keep the corrupted data. JFYI if you do keep "Chapters" in, you could possibly get double chapters with one set being corrupted and the other set being the correct ones.
2) Find all of the manga from MangaOwl.
Note: You can do this by typing "mangaowl" into the search bar of your library
3) Highlight all of them and delete them (and any downloaded chapters) from your library.
Note: You can do this by briefly tapping and holding on one manga, and then continuely tapping on the others you want fixed. You can go flip through other categories without breaking the chain. Be sure to hit the trashcan when you are done.
4) Go to Settings > Advanced and tap on "Clear database".
4.5) You can test random manga that you remember that used to be in your library for peace of mind that it worked.
5) Restore the backup you just made and wait. It could take a long time (20mins+) depending on how big your library is.
6) Update your library; aka more waiting. This will ensure that your manga get the correct online chapter data.
Note: All of the manga should have their total chapters number on the top left on the manga. If numbers do not show up, do not panic. Just open the manga and let it refresh itself. The chapters will show up afterwards.
---‐---------
Both methods will remove all your progress and bookmarks. But hey, you can read now without any no pages found or http/https errors.
Disclaimer: I am not staff. I am just a user. Follow these instructions at your own risk.

**Comment by 25Marc03** [comment]
That's a lot of work…Moreover, can the developer fix it on the next update of Manga Owl? That also hard I have more than 500 Manga connected to Manga Owl, doing it manually as well as the chapters seems difficult 😕

**Comment by D1012** [comment]
Same. Its my main source for my mangas. Moreover I will try this solution. Hoping that developers may find the solution that last.

**Comment by 25Marc03** [comment]
> Same. Its my main source for my mangas. Moreover I will try this solution. Hoping that developers may find the solution that last. 

Please let me know if it works 😄

**Comment by sehee06** [comment]
I try this solution and it works! Actually this is the 2nd time that this kind of issue happened to my library, I have almost 900 mangas saved under Mangaowl and it really frustrates me. Just like what the user said, follow the instructions at your own risk. (And I'm so thankful for these solution!) 👍

**Comment by CVIUS** [comment]
> Short Solution:
> 
> 1. Copy (or memorize) the name of the manga you want fixed
> 2. Remove it (and any downloaded chapters) from your library
> 3. Go to Settings > Advanced and tap on "Clear database".
> 4. Find the manga again on MangaOwl
> 5. Test to see it work and re-add to library
> 6. Repeat until done

I believe migrating manga that is in library from mangaowl to mangaowl after updating the extension will do the same. I'm not quite sure abt this.

**Comment by Hist0ri4** [comment]
It didn't work for me...I hope the developers would fix it soon :( I'm really worried rn. I was so happy when I saw the new update a couple of days ago and after updating I tried opening the chapter. It said like "fix url http" something like that. And later it's back to square one..."no pages found" ://// I hope they'll fix this soon

**Comment by D1012** [comment]
> > Same. Its my main source for my mangas. Moreover I will try this solution. Hoping that developers may find the solution that last.
> 
> Please let me know if it works 😄

It works for me. Hope it last 🤞

**Comment by 25Marc03** [comment]
> It didn't work for me...I hope the developers would fix it soon :( I'm really worried rn. I was so happy when I saw the new update a couple of days ago and after updating I tried opening the chapter. It said like "fix url http" something like that. And later it's back to square one..."no pages found" ://// I hope they'll fix this soon 

Maybe in the next extension update it'll be fixed which I'm hoping. All we need to do is wait and rest for now HAHA.

**Comment by D1012** [comment]
The process was working but it seems for a few minutes or time. Its back to no pages found 😢

**Comment by D1012** [comment]
> I believe I have found the solution for both the no page error and the http/https error. I spent a few hours testing before I had a conclusive answer. The problem is a corruption in the data of the chapters in the library interfering with the correct online data. If you check any manga from MangaOwl that have never been in your library before the errors happened, you will find there are no errors. Simply removing it from your library and adding it back will not work (because the data is still stored on your device). If you have only a few titles from MangaOwl, follow the "Short Solution". If you have a lot of manga (like I do), follow the "Big Library Solution".
> 
> Short Solution:
> 
> 1. Copy (or memorize) the name of the manga you want fixed
> 2. Remove it (and any downloaded chapters) from your library
> 3. Go to Settings > Advanced and tap on "Clear database".
>    Note: For good measure, clear chapter cache and clear cookies as well.
> 4. Find the manga again on MangaOwl
> 5. Test to see it work and re-add to library
> 6. Repeat until done
> 
> Big Library Solution (Edited):
> 
> 1. Back up your library **EXCEPT for "Chapters", "Trackers" and "History"**. You don't want to keep the corrupted data. JFYI if you do keep "Chapters" in, you could possibly get double chapters with one set being corrupted and the other set being the correct ones.
> 2. Find all of the manga from MangaOwl.
>    Note: You can do this by typing "mangaowl" into the search bar of your library
> 3. Highlight all of them and delete them (and any downloaded chapters) from your library.
>    Note: You can do this by briefly tapping and holding on one manga, and then continually tapping on the others you want fixed. You can go flip through other categories without breaking the chain. Be sure to hit the trashcan when you are done.
> 4. Go to Settings > Advanced and tap on "Clear database"
>    Note1: For good measure, clear chapter cache and clear cookies as well.
>    Note2: Before the next step, you can test random manga that you remember that used to be in your library for peace of mind that it worked.
> 5. Restore the backup you just made and wait. It could take a long time (30mins+) depending on how big your library is.
> 6. Update your library; aka more waiting. This will ensure that your manga gets the correct online chapter data.
>    **Note!!!: Before doing step 6, be sure to go to Settings > Library and untoggle "Only update ongoing manga" so you can update your completed manga as well.** I had to refresh many of them one by one....
>    Just found out updating in just the individual categories if there are some missing from the initial update finishes the job just fine. :)
>    Note: All of the manga should have their total chapters number on the top left of the manga. If numbers do not show up, do not panic. Just open the manga and let it refresh itself. The chapters will show up afterward.
> 
> Both methods will remove all your progress and bookmarks. But hey, you can read now without any no pages found or http/https errors. Disclaimer: I am not staff. I am just a user. Follow these instructions at your own risk.
> 
> Edit: I went back after reading the comments and noticed that a lot of my 18+ manga and many of my non-18+ still had the errors. I did another test; replicated the same steps but this time with a backup with no Chapters, Trackers, nor History. I cleared chapter cache and cookies as well. I did a spot check of 45 titles of 18+ and 25 titles of non-18+; All of those seem to all pass (for now). Yeah, I hope they fixed this so none of you have to go through the loops that I had to. Probably spent like 6+ hours on this. I have 800+ titles in Mangaowl so this took forever. Hope this helps anyways. ∠( ᐛ 」∠)＿

Im trying this process for the second time. Hope that this time it's fine. Nonetheless hoping that the developers/staff will look in to this issue.

**Comment by sehee06** [comment]
Yeah, I notice that it has the same error again. Idk if I should try this method again. :(

**Comment by D1012** [comment]
If this will not work again this time, we don't have any choice but to wait for them to look into this and fix this issue. Atleast we tried ☺️

**Comment by sehee06** [comment]
Yeah. And I just noticed their update on discord last Jan 14 that MangaOwl is broken. 😅

**Comment by D1012** [comment]
If its broken, we don't have any other choice then. Hoping for its recovery soon 😅

**Comment by Hist0ri4** [comment]
It's been 2 days and I'm worried...I wanna read cherry blossoms after winter in manga owl but...dang. I hope they'd fix this soon. If the developers read this please fix this 🥺 everytime I see a error in manga owl I feel lost... please fix this ASAP ;-;

**Comment by arkon** [comment]
Merging with #10463
